src/controllers/solidity/index.ts

```diff
@@ -177,12 +177,12 @@ export const solidityAstCommand: Command = {
             args.outputDir,
             fileName,
         ]);
-        const astPath = path.join(args.outputDir, `${path.parse(fileName).name}.ast.json`)
-        let ast = fs.readFileSync(astPath, 'utf-8')
-        if (ast[0] !== '{') { // This is necessary because the original result may not be in JSON format
-            ast = ast.split('\n').slice(4).join('\n')
-        }
-        writeTextFile(astPath, ast)
+   //     const astPath = path.join(args.outputDir, `${path.parse(fileName).name}.ast.json`)
```

Comment:
I think the following lines are no longer needed, I have no idea why I added them sometime

Reply:
Some users asked to save ast file generated by sol.
But in this case the everdev must use additional option to save ast (imho).

Reply:
Everdev has `sol ast` command, so they use it